bundles/org.openhab.binding.mystrom/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mystrom"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="mystromplug">
+		<label>myStrom Smart Plug</label>
+		<description>Controls the myStrom smart plug</description>
+
+		<channels>
+			<channel id="switch" typeId="switch-channel"/>
+			<channel id="power" typeId="power-channel"/>
+			<channel id="temperature" typeId="temperature-channel"/>
+		</channels>
+
+		<config-description>
+			<parameter name="hostname" type="text">
+				<label>Hostname</label>
+				<description>The host name or IP address of the mystrom plug. It may or may not start with the
+					protocol, e.g. in
+					order to use https:// instead of the default http://.</description>
```

Comment:
Is there a reason why you don't use https by default? You could remove the protocol here and add `<context>network-address</context>` to get a free validation, then.

Reply:
done